code/modules/vehicles/mecha/equipment/tools/mining_tools.dm

```diff
@@ -79,7 +80,7 @@
 
 /turf/closed/wall/r_wall/drill_act(obj/item/mecha_parts/mecha_equipment/drill/drill, mob/user)
 	if(drill.drill_level >= DRILL_HARDENED)
-		if(drill.do_after_mecha(src, user, 120 / drill.drill_level))
+		if(drill.do_after_mecha(src, user, 120 / drill.drill_level,))
```

Comment:
?????? is this some 4d chess move of defaulting an arg to null? I thought dm just ignored that?

Reply:
typo there used to be an arg there

Reply:
```suggestion
		if(drill.do_after_mecha(src, user, 120 / drill.drill_level))
```
this is a 4D chess webedit though